sample/TKSetupSamples.m

```diff
@@ -226,9 +231,11 @@ -(void)testRecoverMember {
     // Create a new SDK client with its own keystore to make sure
     // we don't interfere with/use keys used to create the member
     TokenIOBuilder *builder = [self sdkBuilder];
-    builder.keyStore = [[TKInMemoryKeyStore alloc] init];
+    id<TKKeyStore> store = [[TKInMemoryKeyStore alloc] init];
+    builder.cryptoEngineFactory = [TKTokenCryptoEngineFactory factoryWithStore:store
+                                                        useLocalAuthentication:NO];
     TokenIO *tokenIO = [builder buildAsync];
-    __block int prompting = false;
+    __block int prompting = NO;
 
     void (^showPrompt)(NSString *s) = ^(NSString *s) {
         prompting = true;
```

Comment:
Today I learned `NO` thanks to this change. When I look at objective-c things, I see something called `YES` . Maybe it belongs here?

Reply:
You are right. NO and false are just the same, the only different is Apple suggest "NO" in objc as a style.